app/lib/mappers/mapServices.js

```diff
@@ -20,10 +20,59 @@ function getDeliverer(deliverer) {
   return deliverer && deliverer.name;
 }
 
+function mapServiceName(serviceTitle) {
+  const mapTitles = {
```

Comment:
The JSON object created and assigned to `mapTitles` could be in its' own json file and loaded by `require` e.g. if that JSON file was located at `app/lib/resources` (or a more appropriate location) 
`const mapTitles = require('../../resources/servicesTitles')`.
The reason I suggest this is because it isn't going to be changing during the application life cycle and having it in the file with the logic makes the file much larger and more difficult to scan to find the logic.